Source/Core/SerializedMapProjection.js

```diff
@@ -0,0 +1,108 @@
+define([
+        '../ThirdParty/when',
+        './Check',
+        './CustomProjection',
+        './defined',
+        './DeveloperError',
+        './Ellipsoid',
+        './freezeObject',
+        './GeographicProjection',
+        './Proj4Projection',
+        './Rectangle',
+        './WebMercatorProjection'
+    ], function(
+        when,
+        Check,
+        CustomProjection,
+        defined,
+        DeveloperError,
+        Ellipsoid,
+        freezeObject,
+        GeographicProjection,
+        Proj4Projection,
+        Rectangle,
+        WebMercatorProjection) {
+    'use strict';
+
+    var ProjectionType = freezeObject({
+        GEOGRAPHIC : 0,
+        WEBMERCATOR : 1,
+        PROJ4JS : 2,
+        CUSTOM : 3
+    });
+
+    /**
+     * Serializes and Deserializes MapProjections.
```

Comment:
Even though this is private, a little more context about who uses this and why it exists would be helpful here.